website/docs/intro/index.mdx

```diff
@@ -1,59 +1,59 @@
 ---
 sidebar_position: 1
 description: |-
-  OpenTF is an infrastructure as code tool that lets you build, change, and version cloud and on-prem resources safely and efficiently.
+  OpenTofu is an infrastructure as code tool that lets you build, change, and version cloud and on-prem resources safely and efficiently.
 ---
 
-# What is OpenTF?
+# What is OpenTofu?
 
-OpenTF is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. OpenTF can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
+OpenTofu is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. OpenTofu can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
 
-## How does OpenTF work?
+## How does OpenTofu work?
 
-OpenTF creates and manages resources on cloud platforms and other services through their application programming interfaces (APIs). Providers enable OpenTF to work with virtually any platform or service with an accessible API.
+OpenTofu creates and manages resources on cloud platforms and other services through their application programming interfaces (APIs). Providers enable OpenTofu to work with virtually any platform or service with an accessible API.
 
-![OpenTF creates and manages cloud platforms and services through their APIs](/img/docs/intro-opentf-apis.png)
+![OpenTofu creates and manages cloud platforms and services through their APIs](/img/docs/intro-tofu-apis.png)
```

Comment:
The current image file is https://github.com/opentofu/opentofu.org/blob/main/static/img/docs/intro-opentf-apis.png

I think we should rename the file to the full `opentofu` name